modules/end-user-guide/pages/using-a-git-provider-access-token.adoc

```diff
@@ -116,6 +116,20 @@ __<Secret_prepared_in_step_5>__
 EOF
 ----
 ====
+. Azure DevOps Server. If you are using Azure DevOps Server, you must also modify the xref:mounting-git-configuration.adoc[workspace's
+gitconfig] with the following section:
++
+```gitconfig
```

Comment:
is this a code block? just double-checking because it's not in asciidoc

Reply:
sorry, that's still not it. this is what asciidoc code blocks look like:

[source, code]
----
`stuff`
----

Reply:
gosh darnit

`[source, code]
----
`stuff`
----`